packages/adblocker-benchmarks/Makefile

```diff
@@ -10,6 +10,7 @@ requests.json:
 
 ./node_modules/ubo-snfe:
 	make -C ./blockers/ublock install-nodejs clean
+	npx rollup ./node_modules/ubo-snfe/main.js --file ./node_modules/ubo-snfe/bundle.min.cjs --format cjs --context global --plugin terser
```

Comment:
If you use `npx` then I guess you do not need the devDependencies.

Reply:
Thanks, I did not know this. Let me give it a try.

Reply:
I think `devDependencies` is still needed, because otherwise `npx` wouldn't know where to get the executables and the plugins, but I can try to confirm this.

Reply:
Now I understand what you meant: rollup.js is already installed so `npx` can just use it. Yeah, but we're going to remove the dependencies from ubo-snfe so it won't already be installed. We'll need `devDependencies` here.